src/java/org/apache/cassandra/net/async/NettyFactory.java

```diff
@@ -113,6 +118,7 @@
                                         "MessagingService-NettyAcceptor-Threads", false);
         inboundGroup = getEventLoopGroup(useEpoll, FBUtilities.getAvailableProcessors(), "MessagingService-NettyInbound-Threads", false);
         outboundGroup = getEventLoopGroup(useEpoll, FBUtilities.getAvailableProcessors(), "MessagingService-NettyOutbound-Threads", true);
+        streamingGroup = getEventLoopGroup(useEpoll, FBUtilities.getAvailableProcessors(), "Streaming-Netty-Threads", false);
```

Comment:
This is a benchmarking question, but there probably shouldn't be an inbound group and outbound group for regular messages either. Ideally it would be thread per core and load balanced, but Netty doesn't load balance so the right answer for how many event loop groups to have is going to have to come from measuring and might be workload specific.